airsonic-main/src/main/java/org/airsonic/player/controller/HomeController.java

```diff
@@ -142,7 +142,11 @@ protected ModelAndView handleRequestInternal(HttpServletRequest request) throws
         List<Album> result = new ArrayList<>();
         for (MediaFile mediaFile : ratingService.getHighestRatedAlbums(offset, count, musicFolders)) {
             Album album = createAlbum(mediaFile);
-            album.setRating((int) Math.round(ratingService.getAverageRating(mediaFile) * 10.0D));
+            Double averageRating = ratingService.getAverageRating(mediaFile);
+            if (averageRating == null) {
```

Comment:
Isn't it better to assume `averageRating = 0` in case it doesn't exist?

Reply:
The `continue` ensures that the album's rating isn't modified. You prefer to set the rating to zero?